compiler/optimizer/VPHandlers.cpp

```diff
@@ -4382,7 +4386,11 @@ void addValidRangeBlockOrGlobalConstraint (OMR::ValuePropagation *vp,
 
    if (pLow > pHigh)
       {
+#if defined (OMRZTPF)
+           VPHswap (pLow, pHigh);
```

Comment:
There is too much indentation in this line.

Reply:
Thanks... I hope I updated/squashed the changes correctly!